test/harness.spec.js

```diff
@@ -58,6 +58,29 @@ describe('Harness tests', () => {
         patient_age_in_years: '19',
       });
     });
+
+    it('setNow works with DateTime', async () => {
+      const t = DateTime.now();
+      await harness.setNow(t);
+      expect(harness.getNow()).to.equal(t.toMillis());
+    });
+
+    it('flush works with Duration', async () => {
+      await harness.setNow('2000-01-01');
+      const d = Duration.fromISO('P5Y3M'); // 5 years, 3 months
+      await harness.flush(d);
+      const now = await harness.getNow();
+      expect(new Date(now).toUTCString()).to.include('Fri, 01 Apr 2005 00:00:00');
+    });
+
+    it('flush accounts for DST', async () => {
```

Comment:
nit: i like putting the issue number in the name of the test. tests can read like user requirements, and linking to the issue can provide deep back-story with little/no effort for the test author

`'#20 - flush accounts for DST'`

Reply:
Done